src/Microsoft.ML.Data/DataLoadSave/TransformWrapper.cs

```diff
@@ -148,11 +152,13 @@ IRowToRowMapper ITransformer.GetRowToRowMapper(DataViewSchema inputSchema)
     /// <summary>
     /// Estimator for trained wrapped transformers.
     /// </summary>
-    public abstract class TrainedWrapperEstimatorBase : IEstimator<TransformWrapper>
+    internal abstract class TrainedWrapperEstimatorBase : IEstimator<TransformWrapper>
```

Comment:
This is a funny one. I was able to remove all subclasses of it, but some notes I read elsewhere led me to suspect there may be more than this than meets the eye. Started an issue about this for #2841. (Though that does not touch the public surface and is therefore inessential.)